create_artifact.py

```diff
@@ -0,0 +1,37 @@
+"""
+This is the code for creating an artifact through wandb.
+"""
+import wandb
+from fastai.vision.all import *
+
+import zipfile as zf
+# Change zip file name to the folder used to save screenshots
+files = zf.ZipFile('UE_Screenshots.zip', 'r')
+files.extractall('UE_Screenshots')
+files.close()
+
+run = wandb.init(
+    # Change project and notes to be specific to the artifact you're creating
+    project="Multirun-testing-4K",
```

Comment:
We should come up with a standard naming scheme. Will one of you all look into it? @daisy-abbott @anjalinugg4 @chauvuha @framor81